JS.md

```diff
@@ -522,12 +520,8 @@ or `null`, throw a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-erro
 
 Let `i` be the result of [`ToNonWrappingUint32`](#tononwrappinguint32)(`index`).
 
-If `v` is an [Exported Function Exotic Object](#exported-function-exotic-objects):
-* Set the `i`th element of `T.[[Table]]` to the `v.[[FunctionIndex]]`th function
-  in the module's [function index space](Modules.md#function-index-space).
```

Comment:
Actually, looking at it now, I'm confused about the purpose of T.[[Table]], which doesn't seem to be used for anything. Also, the representation in the spec changed: a table no longer contains indices but actually a function values as well. Is this just essentially duplicating the same thing as T.[[Values]] now? Should I try to get rid of one?